### ISSUE TITLE
Prokka fails for rnaspades

### ISSUE THREAD

**Issue by emnilsson** [LNUc-EEMiS/metatdenovo#98]
## Check Documentation

I have checked the following places for your error:

- [ ] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
- [ ] [nf-core/metatdenovo pipeline documentation](https://nf-co.re/metatdenovo/usage)

## Description of the bug

When running `nextflow run . -profile docker,test --assembler rnaspades --orf_caller prokka`, `prokka` fails since the contig names are too long.

## Steps to reproduce

Steps to reproduce the behaviour:

1. Command line: `nextflow run . -profile docker,test --assembler rnaspades --orf_caller prokka`
2. See error: 
`  [07:17:58] Contig ID must <= 37 chars long: NODE_1_length_638965_cov_4.730899_g0_i0
    [07:17:58] Please rename your contigs OR try '--centre X --compliant' to generate clean contig names.`

## Log files

Have you provided the following extra information/files:

- [ ] The command used to run the pipeline
- [ ] The `.nextflow.log` file 

## System

- Hardware: 
- Executor: 
- OS: 
- Version 

## Nextflow Installation

- Version: 

## Container engine

- Engine: docker
- version:

**Comment by erikrikarddaniel** [comment]
Seems like the simplest would be to add `--centre X --compliant` to `modules.config`? (I've used this before, but had forgotten why.)

**Comment by emnilsson** [comment]
Cannot reproduce on the "code go thru"-pr branch, I'm getting new errors there:
```
No such variable: id
 -- Check script './workflows/../subworkflows/local/prokka_subsets.nf' at line: 32 or see '.nextflow.log' file for more details
No such file: emniab
No such variable: id
 -- Check script './workflows/../subworkflows/local/prokka_subsets.nf' at line: 26 or see '.nextflow.log' file for more details
No such variable: id
 -- Check script './workflows/../subworkflows/local/prokka_subsets.nf' at line: 20 or see '.nextflow.log' file for more details
```

It seems caused by:
`contigs.map{ [ id:"${it[0].id}.prokka" ] }` being interpreted as my user name? Something wrong on my end or in general?

**Comment by danilodileo** [comment]
We fixed the issue in PR #102
